k8sutils/pkg/consts/consts.go

```diff
@@ -1,14 +1,30 @@
 package consts
 
+type CollectorType string
+
+const (
+	// Cluster collector is responsible for exporting observability data from the cluster.
+	ClusterCollector CollectorType = "cluster"
+	// Node collector is receiving data from different instrumentation SDKs in the same node.
+	NodeCollector    CollectorType = "node"
+)
```

Comment:
I wonder, because we already have this [here](https://github.com/odigos-io/odigos/blob/e7d6f717a32c1407ecccff4715f541acf5ff1169/api/odigos/v1alpha1/collectorsgroup_types.go#L28):

```
// +kubebuilder:validation:Enum=CLUSTER_GATEWAY;NODE_COLLECTOR
type CollectorsGroupRole string

const (
	CollectorsGroupRoleClusterGateway CollectorsGroupRole = "CLUSTER_GATEWAY"
	CollectorsGroupRoleNodeCollector  CollectorsGroupRole = "NODE_COLLECTOR"
)
```

and this [here](https://github.com/odigos-io/odigos/blob/e7d6f717a32c1407ecccff4715f541acf5ff1169/instrumentor/controllers/instrumentationdevice/common.go#L59):

```js
if (cg.Spec.Role == odigosv1.CollectorsGroupRoleNodeCollector || cg.Spec.Role == "DATA_COLLECTION") && cg.Status.Ready {
```

Should we use the same names? and should we share the enum everywhere? now or later